creator-node/src/routes/healthCheck.js

```diff
@@ -108,7 +108,8 @@ module.exports = function (app) {
       available: _formatBytes(available),
       total: _formatBytes(total),
       usagePercent: `${usagePercent}%`,
-      maxUsagePercent: `${maxUsagePercent}%`
+      maxUsagePercent: `${maxUsagePercent}%`,
+      path
```

Comment:
ik this was already called `path` within the code but it would be better if it was just explicitly `storagePath` too

Reply:
fixed